Source/Urho3D/CSharp/Managed/Math/Matrix3.cs

```diff
@@ -62,6 +62,28 @@ public Matrix3(IReadOnlyList<float> data)
             M22 = data[8];
         }
 
+        /// <summary>
+        /// Construct from an angle (in degrees) and axis.
+        /// </summary>
+        public Matrix3(float angle, Vector3 axis)
```

Comment:
I don't like that we now have two identical algorithms in different places.
`Quaternion(angle, axis).ToMatrix3()` and `Matrix3(angle, axis)` are copypaste basically. Any way we can get rid of it?

Reply:
There is no Quaternion.ToMatrix3() in c#

Reply:
https://github.com/rbfx/rbfx/blob/master/Source/Urho3D/CSharp/Managed/Math/Quaternion.cs#L422
There totally is

Reply:
It isn't called ToMatrix3. Naming consistency isn't great :-(

Reply:
I don't always remember exact function names. It's called the same way in C++ and in C# tho, so it's good enough

Reply:
Fixed